docs/conf.py

```diff
@@ -87,7 +87,7 @@
 # a list of builtin themes.
 #
 # html_theme = 'alabaster'
-html_theme = 'sphinx_rtd_theme'
+html_theme = 'alabaster'
```

Comment:
Why changing theme?
If you want to change theme why not commenting

    html_theme = 'sphinx_rtd_theme

and commenting out

html_theme = 'alabaster'

Reply:
Uups, that was an accident and shouldn't be included into this commit. Will change it.